src/controllers/authenticationController.ts

```diff
@@ -109,10 +103,16 @@ export default class AuthenticationController {
         newProfile
       );
       if (createProfileResponse.kind !== "IResponseSuccessJson") {
-        return createProfileResponse;
+        // TODO: Must be extended IAuthenticationController to support IResponseErrorTooManyRequests?
+        return createProfileResponse.kind === "IResponseErrorTooManyRequests"
+          ? ResponseErrorInternal("To many requests")
```

Comment:
Why this changes ?

```suggestion
          ? ResponseErrorInternal("Too many requests")
```

Reply:
Because the actual implementation of `IAuthenticationController` doesn't support  `IResponseErrorTooManyRequests` as accepted response for the `acs` method.